src/main/java/envoy/server/processors/package-info.java

```diff
@@ -0,0 +1,9 @@
+/**
+ * This package contains all processors that can receive an object from the
+ * client and interact with it.
```

Comment:
Missing double `<br>` tags.

Also:
```suggestion
 * Envoy Client and interact with it.
```